pom.xml

```diff
@@ -457,9 +504,50 @@
                   <version>0.53</version> <!-- {x-version-update;org.apidesign.javadoc:codesnippet-doclet;external_dependency} -->
                 </docletArtifact>
                 <additionalOptions>
-                  <additionalOption>-maxLineLength 120</additionalOption>
-                  <additionalOption>-snippetpath ${project.basedir}/src/samples/java</additionalOption>
-                  <additionalOption>-suppressMissingLinkWarnings</additionalOption>
+                  <additionalOption>-maxLineLength 120
+                    -suppressMissingLinkWarnings
+                    -snippetpath ${project.basedir}/sdk/agrifood/azure-verticals-agrifood-farming/src/samples/java
```

Comment:
Does `snippetpath` support `**`?

Reply:
Unfortunately, snippetpath doesn't support wildcards in path. I had filed an issue for this - https://github.com/jtulach/codesnippet4javadoc/issues/15